24_clustering.Rmd

```diff
@@ -18,7 +18,7 @@ features/samples that share pattern.  For example, with clustering, we
 can find group of samples that share similar community
 composition. There are multiple clustering algorithms available.
 
-## Introduction to bluster package
+## Introduction to bluster package and wrapper
```

Comment:
Another stylistic suggestion - I would generally avoid mentioning individual packages in titles. Instead I would focus on the conceptual idea. Here, an alternative title could be like: "Custom tools"